extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -33,6 +33,12 @@ const POSITION_FIXED_TAG_WHITELIST = {
   'AMP-STICKY-AD': true,
 };
 
+/** @const These tags are valid ad containers */
+const AMP_AD_CONTAINERS = {
```

Comment:
We should probably think of `position: fixed` as the container, so lightboxes would be in this list as well.